### ISSUE TITLE
Update CoC to explain it's "Paused" status to avoid confusion. 

### ISSUE THREAD

**Issue by preoctopus** [todogroup/opencodeofconduct#87]
Since it's been agreed from #84 that the CoC is not ready for use in it's current state. I won't reiterate the current issues as @bkeepers mentions them in the linked issue, as well as others mentioned in the "closed" issues/PRs regarding a range of problematic statements in it's current form. 

Can we get an update to this document's readme.md explaining the pending review, preferably with an ETA? 

There is still a [blog post](https://github.com/blog/2039-adopting-the-open-code-of-conduct) on github's blog pointing people towards this as if it were ready to use. I'm not sure why that is still there, but it should be removed/updated as well as to not mislead interested people.

**Comment by davblayn** [comment]
@preoctopus There's been a minor update to the original blog post, adding the following:

>The Open Code of Conduct does not apply to all activity on GitHub, and it does not alter GitHub's Terms of Service.

**Comment by preoctopus** [comment]
@ghost Excellent, though that doesn't explain the absence of a "paused" disclaimer on the project. 

@bkeepers Any chance of an update on why the discriminatory clauses remain? Or at the very least an explanation on when the might apply? Or is there no interest in feedback on this anymore?

**Comment by vgturtle127** [comment]
#97 should fix the problem. :smile: 

@bkeepers @ammeep @Addvilz @caniszczyk

**Comment by giulioprisco** [comment]
Yes, please update CoC to explain it's "Paused" status to avoid confusion. As things stand it seems that Github endorses the original text inclusing the "reverse" bit that was rejected by the majority of participants in the discussion.
